3634/3/app/views/restaurants/show.html.slim

```diff
@@ -0,0 +1,32 @@
+.container.w-75.p-4.text-light
+  p.text-right
+    a{href="/" class="text-light"} BACK
+  h1.m-4.text-center = @restaurant.name
+  .border.m-4
+    p.m-3.text-justify = @restaurant.description
+    p.m-3.text-center
+      | Latitude:&nbsp;&nbsp;
+      | #{@restaurant.latitude} &nbsp;&nbsp;
+      | Longitude:&nbsp;&nbsp;
+      | #{@restaurant.longitude}
+    p.m-3.text-center
+      - if @raiting
+          | Average mark: #{@raiting.round}
```

Comment:
`.round` call need to move to the controller. It is part of the logic. And according to MVC pattern, a logic locate in a controller.